hermeto/core/package_managers/bundler/main.py

```
@@ -122,6 +123,17 @@ def _resolve_bundler_package(
                 concurrency_limit=get_config().runtime.concurrency_limit,
             )
         )
+        for dep in (d for d in dependencies if isinstance(d, GemDependency)):
+            if dep.checksum is None:
+                log.warning(
+                    "No checksum found for %s-%s, skipping verification", dep.name, dep.version
+                )
+                continue
+            else:
```

Ruff (0.15.21)

[warning] Unnecessary `else` after `continue` statement

Remove unnecessary `else`

(RET507)